client/components/.eslintrc

```diff
@@ -3,10 +3,9 @@
     "browser": true,
     "node": true
   },
-  "settings": {
-    "ecmascript": 6,
-    "jsx": true
-  },
+  "plugins": [
+    "react"
+  ],
```

Comment:
When you add the react plugin you need to explicitly turn on the react rules or they will not apply by default.